config.example.yaml

```diff
@@ -47,3 +47,5 @@ destinations:
       max_mute_minutes: 20160 # 14 days
       # Optional. If not specified the alert_key will default to this value.
       alert_key: "This alert was created via the alerts framework"
+      retries: 3
+      datadog_retry_base_delay: 0.3
```

Comment:
since it's defined for the datadog destination, wouldn't `datadog_` be redundant? I think it should just be `retry_base_delay` or `retry_delay_seconds`